nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -844,7 +844,7 @@ nva:JournalReview a rdfs:Class ;
   rdfs:subClassOf nva:Journal .
 
 nva:PublicationDate a rdfs:Class ;
-  rdfs:comment "" ;
+  rdfs:comment "When the publication was first published in any system or form" ;
```

Comment:
This is a thankless task since we need to agree…and we will all have very different ideas about what words mean…it should have been done at the beginning and maintained. Thanks for picking this up!